### PR TITLE
Update evaluation scripts

### PR DESCRIPTION
**Pull request by myron**
Update evaluation scripts to most recent.

### CODE REVIEW

**Comment by mingxin-zheng** [comment]
Hi @myron , please fix the errors in:
https://github.com/Project-MONAI/VLM/actions/runs/11493176280/job/31988445625?pr=37

Thanks!

File: m3/eval/eval_chexpert_class.slurm [hunk 1]
```diff
@@ -7,7 +7,7 @@
 #SBATCH --partition=interactive,interactive_singlenode,grizzly,polar,polar2,polar3,polar4
 #SBATCH --exclusive
 #SBATCH --dependency=singleton
-#SBATCH -A healthcareeng_monai
+#SBATCH -A monai
```
Comment (holgerroth) [inline m3/eval/eval_chexpert_class.slurm:10]: Can we use `#SBATCH - A <YOUR_SLURM_ACCOUNT>` in these scripts to avoid confusion.

**Review by holgerroth — COMMENTED** [review]
Added one comment. Otherwise looks good.

File: m3/eval/eval_chexpert_class.slurm (hunk 1 above)
Comment (myron) [reply m3/eval/eval_chexpert_class.slurm]: ok, I moved it to env var

**Review by holgerroth — APPROVED** [review]
Looks good!

**Pull request merged**